src/App.js

```diff
@@ -3,12 +3,70 @@ import Mockman from "mockman-js";
 import { ToastContainer, } from 'react-toastify';
 import {HomePage} from "Pages/HomePage/HomePage";
 import {ProfilePage} from "Pages/ProfilePage/ProfilePage";
+import {Page404} from "Pages/Page-404/Page404";
+import {Route,Routes,Outlet, useLocation} from "react-router-dom";
+import { useEffect } from "react";
+import {useDispatch, useSelector} from "react-redux";
+import { checkToken} from "Redux/Reducers-Redux/authSlice";
+import PrivateRoute from "Components/CustomRoute/PrivateRoute";
+import RestrictedRoute from "Components/CustomRoute/RestrictedRoute";
+import {getAllUsers} from "Redux/Reducers-Redux/usersSlice";
+import {editPostContent, getAllPosts,getBookMarks} from "Redux/Reducers-Redux/postsSlice";
+import BookMarkPage from "Pages/BookMarkPage/BookMarkPage";
```

Comment:
Suggestion : 

- To import files from the pages and components folder, you can create 1 index.js file in each folder, and export all files 
   from there itself .

- It will help with readability and clean code.

- It will help with debugging , if you have made some error with files import-export, then you will have to check only in the index.js file.